packages/server/customer-os-postgres-repository/repository/flow_transitions_registry_repository.go

```diff
@@ -40,6 +40,7 @@ func (r *flowTransitionsRegistryRepository) GetUniqueFromNodes(ctx context.Conte
 
 	var uniqueFromNodes []FromNodeRecord
 	err := r.gormDb.WithContext(ctx).
+		Table("flow_transitions_registry").
```

Comment:
Use Model function

Reply:
db.Model(&FlowTransitionsRegistry{}).

instead of Table...?